<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QDialog" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{border-image: url(:/new/prefix1/pic/city.jpg);}</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountLB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/pic/acc.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLE">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>得意黑</family>
     <pointsize>22</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>欢迎登录管理员系统</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdLE">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eyeButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/pic/passon.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="solfkeyui" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>600</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236,142, 158);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountLE">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="placeholderText">
    <string>请输入手机号码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwdLB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/pic/lock.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="captCha">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>验证码登录</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>solfkeyui</class>
   <extends>QWidget</extends>
   <header location="global">solfkeyui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
